src/pages/publication/references_tab/BookReferenceForm.tsx

```diff
@@ -68,25 +68,31 @@ const BookReferenceForm: FC = () => {
       <Field name={BookFieldNames.PUBLISHER}>
         {({ field: { name, value } }: any) => (
           <>
-            <PublicationChannelSearch
-              clearSearchField={value === emptyPublisher}
-              label={t('references.publisher')}
-              publicationTable={PublicationTableNumber.PUBLISHERS}
-              setValueFunction={inputValue => setFieldValue(name, inputValue ?? emptyPublisher)}
-            />
-            {value.title && (
-              <JournalPublisherRow
-                hidePublisher
+            <div data-testid="autosearch-publisher">
```

Comment:
ser ut som man kan legge `data-testid` på komponentene direkte, så slipper vi mange unødvendige divs: https://dev.to/sama/testing-material-ui-form-components-1cnh